Integration tests for DigiD have wrong class name

... and renaming this breaks things.

Additionally, I'm getting local test failures on the master branch.

- [ ] `src/openforms/authentication/contrib/eherkenning/tests/test_signicat_integration.py` -> `class SignicatEHerkenningIntegrationTests` should be `class SignicatDigiDIntegrationTests`, but doing so breaks the VCR casettes
- [ ] `src/manage.py test openforms.authentication.contrib.digid.tests.test_signicat_integration` fails locally, on the master branch
  
  When I'm using `openforms.conf.dev` settings module. It also creates the cassette dir: `src/openforms/authentication/contrib/digid/tests/data/vcr_cassettes/SignicatEHerkenningIntegrationTests/`.

  It looks like things started failing locally because of #3394

```
======================================================================
ERROR: test_raising_loa_requirements_fails_in_flight_authentications (openforms.authentication.contrib.eherkenning.tests.test_signicat_integration.SignicatEHerkenningIntegrationTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.10/unittest/mock.py", line 1379, in patched
    return func(*newargs, **newkeywargs)
  File "/home/bbt/code/open-forms/src/openforms/authentication/contrib/eherkenning/tests/test_signicat_integration.py", line 503, in test_raising_loa_requirements_fails_in_flight_authentications
    session.request(method_2, redirect_url_2, data=form_values_2).ok
  File "/home/bbt/.virtualenvs/open-forms/lib/python3.10/site-packages/log_outgoing_requests/log_requests.py", line 52, in new_request
    return self._lor_initial_request(*args, **kwargs)
  File "/home/bbt/code/open-forms/src/openforms/setup.py", line 133, in new_request
    return self._original_request(*args, **kwargs)
  File "/home/bbt/.virtualenvs/open-forms/lib/python3.10/site-packages/requests/sessions.py", line 575, in request
    prep = self.prepare_request(req)
  File "/home/bbt/.virtualenvs/open-forms/lib/python3.10/site-packages/requests/sessions.py", line 486, in prepare_request
    p.prepare(
  File "/home/bbt/.virtualenvs/open-forms/lib/python3.10/site-packages/requests/models.py", line 368, in prepare
    self.prepare_url(url, params)
  File "/home/bbt/.virtualenvs/open-forms/lib/python3.10/site-packages/requests/models.py", line 439, in prepare_url
    raise MissingSchema(
requests.exceptions.MissingSchema: Invalid URL '/cookies/accept/': No scheme supplied. Perhaps you meant https:///cookies/accept/?

======================================================================
FAIL: test_resuming_a_suspended_submission (openforms.authentication.contrib.eherkenning.tests.test_signicat_integration.SignicatEHerkenningIntegrationTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.10/unittest/mock.py", line 1379, in patched
    return func(*newargs, **newkeywargs)
  File "/home/bbt/code/open-forms/src/openforms/authentication/contrib/eherkenning/tests/test_signicat_integration.py", line 274, in test_resuming_a_suspended_submission
    self.assertEqual(acs_response.resolver_match.view_name, "core:form-detail")
AssertionError: 'submissions:resume' != 'core:form-detail'
- submissions:resume
+ core:form-detail
```

--- a/src/openforms/authentication/contrib/digid/tests/test_signicat_integration.py
+++ b/src/openforms/authentication/contrib/digid/tests/test_signicat_integration.py
@@ -37,7 +37,7 @@ SIGNICAT_BROKER_BASE = furl("https://maykin.pre.ie01.signicat.pro/broker")
 )
 @temp_private_root()
 @override_settings(COOKIE_CONSENT_ENABLED=False)
-class SignicatEHerkenningIntegrationTests(vcr.unittest.VCRMixin, TestCase):
+class SignicatDigiDIntegrationTests(vcr.unittest.VCRMixin, TestCase):
     """Test using Signicat broker.
 
     Instead of mocking responses. We do real requests to a Signicat test environment
@@ -93,8 +93,8 @@ class SignicatEHerkenningIntegrationTests(vcr.unittest.VCRMixin, TestCase):
     def _get_vcr_kwargs(self):
         kwargs = super()._get_vcr_kwargs()
         # https://vcrpy.readthedocs.io/en/latest/usage.html#record-modes
-        # kwargs["record_mode"] = "none"  # in CI
-        kwargs["record_mode"] = "once"  # in dev
+        kwargs["record_mode"] = "none"  # in CI
+        # kwargs["record_mode"] = "once"  # in dev
         return kwargs
 
     def setUp(self):

